spec/forms/registration_form_spec.rb

```diff
@@ -0,0 +1,120 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe RegistrationForm do
+  describe '#save' do
+    context 'given valid user input' do
+      it 'returns true' do
+        application ||= Fabricate(:application)
```

Comment:
We can simplify this assignment like this, please update the spec below as well. 

The `||=` is not necessary. Reference: http://www.rubyinside.com/what-rubys-double-pipe-or-equals-really-does-5488.html


```suggestion
        application = Fabricate(:application)
```